aufs2 kbuild patch for linux-2.6.28

diff --git a/fs/Kconfig b/fs/Kconfig
index 522469a..6ef091b 100644
--- a/fs/Kconfig
+++ b/fs/Kconfig
@@ -1122,6 +1122,8 @@ config UFS_DEBUG
 	  Y here.  This will result in _many_ additional debugging messages to be
 	  written to the system log.
 
+source "fs/aufs/Kconfig"
+
 endmenu
 
 menuconfig NETWORK_FILESYSTEMS
diff --git a/fs/Makefile b/fs/Makefile
index d9f8afe..1f206a2 100644
--- a/fs/Makefile
+++ b/fs/Makefile
@@ -122,3 +122,4 @@ obj-$(CONFIG_HPPFS)		+= hppfs/
 obj-$(CONFIG_DEBUG_FS)		+= debugfs/
 obj-$(CONFIG_OCFS2_FS)		+= ocfs2/
 obj-$(CONFIG_GFS2_FS)           += gfs2/
+obj-$(CONFIG_AUFS_FS)           += aufs/
diff --git a/include/linux/Kbuild b/include/linux/Kbuild
index e531783..14ecb7c 100644
--- a/include/linux/Kbuild
+++ b/include/linux/Kbuild
@@ -34,6 +34,7 @@ header-y += atmppp.h
 header-y += atmsap.h
 header-y += atmsvc.h
 header-y += atm_zatm.h
+header-y += aufs_type.h
 header-y += auto_fs4.h
 header-y += ax25.h
 header-y += b1lli.h
